### PR TITLE
Kctl plugin

### PR DESCRIPTION
**Pull request by zwwhdls**
1. `kubectl jfs get pod` : get all app pods using juicefs pvc
2. `kubectl jfs get mount `: get all juicefs mount pods
3. `kubectl jfs debug pod <name>`: debug app pod using juicefs pvc
4. `kubectl jfs oplog <name>`: get oplog of mount pod
5. `kubectl jfs warmup <name> [<subpath>]`: warmup in mount pod
6. `kubectl jfs pvc/pv`: get all juicefs pvc/pv
7. `kubectl jfs debug pvc/pv <name>`: debug juicefs pvc/pv

### CODE REVIEW

File: cmd/plugin/debug.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var debugCmd = &cobra.Command{
+	Use:                   "debug <resource> <name>",
+	Short:                 "Debug the pod/pv/pvc which is using juicefs",
+	DisableFlagsInUseLine: true,
+	Example: `  # debug the pod which is using juicefs pvc
+  kubectl juicefs debug po <pod-name> -n <namespace>
```
Comment (zxh326) [inline cmd/plugin/debug.go]: juicefs or jfs?

File: cmd/plugin/oplog.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		return fmt.Errorf("please specify the mount pod name")
+	}
+
+	podName := args[0]
+	if !strings.HasPrefix(podName, "juicefs-") {
+		return fmt.Errorf("pod %s is not juicefs mount pod\n", podName)
+	}
+	var pod *corev1.Pod
+
+	if pod, err = clientSet.CoreV1().Pods("kube-system").Get(context.Background(), podName, metav1.GetOptions{}); err != nil {
```
Comment (zxh326) [inline cmd/plugin/oplog.go]: make namespace configurable?

**Comment by zxh326** [comment]
功能好像很全，是不是更适合单独拉一个 repo 出来，比如叫 kubectl-jfs-debug

然后是不是可以收录进 https://github.com/kubernetes-sigs/krew-index

用户可以直接 k krew install？

**Comment by zwwhdls** [comment]
@zxh326 可以

**Pull request closed without merging**